src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,26 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private final List<Bulldozer> bulldozers = new ArrayList<>();
+
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer1 = new Bulldozer(15_000,"Mini Bulldozer");
```

Comment:
Don't name values with numbers. Change it in all cases
```suggestion
        Bulldozer firstBulldozer = new Bulldozer(15_000,"Mini Bulldozer");
```